### DISCUSSION TITLE
Very Confused

### DISCUSSION THREAD

**Discussion by ghost** [FabricMC/fabric-api#2165] [Player support] [answered]
Okay, I made a github account just to make this post. I downloaded the Dreamcraft FABRIC 1.18.1 modpack to play on my server with friends today. I launched it perfectly fine, everything loads good, but when I go to join my server, I get this: 
![image](https://user-images.githubusercontent.com/104487392/165458802-fd452b42-bfad-411f-94f5-e8ac1ac50ba0.png)

I have no idea what it means, and I've tried reading through the latest.log file and I'm not getting anywhere with that. I'll leave that here too. 

[latest.log](https://github.com/FabricMC/fabric/files/8569615/latest.log)

If anybody knows what the heck is going on and could help, that would be amazing. Thank you! <3

If you wanna reach me faster in response to this you can DM me on discord (aeron_noxx#0629)

**Comment by SirEnder125** [comment]
I'm guessing that
```
[00:38:49] [Render thread/INFO]: No Optifine!
[00:38:49] [Render thread/INFO]: No Vivecraft!
```
and what follows may have a great deal to do with the problem...

**Comment by Technici4n** [comment, marked as answer]
It looks like you have a mod version mismatch. For exemple, your client appears to be missing blocks from Artifality. Make sure that you downloaded the same pack as the one you put on the server.

**Reply by ghost** [reply]
I don't know how it's a mismatch, I just downloaded the modpack straight from GDLauncher, and then on BisectHosting searched Dreamcraft from their list of modpacks and it seems to be the same. How do I know what the difference is besides Artifality? (Is it basically saying I don't have Artifality?)

**Reply by ghost** [reply]
i didnt have artifality. kms
